addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-spider.html

```diff
@@ -39,6 +37,13 @@ <H2>YAML</H2>
       sendRefererHeader:               # Bool: Whether the spider will send the referer header, default: true
       threadCount:                     # Int: The number of spider threads, default: 2
       userAgent:                       # String: The user agent to use in requests, default: '' - use the default ZAP one 
+    tests:
```

Comment:
I dont think we can expect the users to know the ZAP stats or which ones would be most relevant. So for both the help and the min/max data I think we should include the tests we think are most appropriate (if any) - so in this case it could be something like:
```
- name: 'Check number of URLs found'   # String: Name of the test, default: statistic + operator + value
  type: 'stats'                        # String: Type of test, supported: 'stats'
  statistic: 'spider.urls.added'       # String: Name of an integer / long statistic, supported: 'spider.urls.added'
  operator: '<'                        # String ['==', '!=', '&gt;=', '&gt;', '&lt;', '&lt;=']: Operator used for testing
  value: 100                           # Int: Value to compare statistic against, change this to the min number of urls you expect to find
  onFail: 'ignore'                     # String [warn, error, ignore]: Change this to 'warn' or 'error' for the test to take effect
```